student.rb

```diff
@@ -1,9 +1,16 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    add_to_classroom
```

Comment:
Great job @AbdelazizSoliman ✋

- [ ] You have done great implementing the requested changes. However, you do not need to call this `add-to-classroom` method in the constructor. Could you maintain this method separate from the constructor?
![add-to-classroom](https://github.com/AbdelazizSoliman/OOP-school-library/assets/63917725/76d94db2-a3f8-465f-8913-fdd1d44dae65)